modules/kde/hm.nix

```
@@ -251,49 +295,53 @@ let
   # might be installed, and look there. The ideal solution would require
   # changes to KDE to make it possible to update the wallpaper through
   # config files alone.
-  activator = pkgs.writeShellScriptBin "stylix-set-kde-wallpaper" ''
+  activator' = pkgs.writeShellScriptBin "stylix-activate-kde" ''
     set -eu
-    global_path() {
+    get_exe() {
       for directory in /run/current-system/sw/bin /usr/bin /bin; do
         if [[ -f "$directory/$1" ]]; then
           printf '%s\n' "$directory/$1"
           return 0
         fi
       done
-
+      echo "Skipping `$1`: command not found"
       return 1
     }
 
-    if wallpaper_image="$(global_path plasma-apply-wallpaperimage)"; then
+    if wallpaper_image="$(get_exe plasma-apply-wallpaperimage)"; then
       "$wallpaper_image" "${themePackage}/share/wallpapers/stylix"
-    else
-      echo "Skipping plasma-apply-wallpaperimage: command not found"
     fi
 
-    if look_and_feel="$(global_path plasma-apply-lookandfeel)"; then
+    if look_and_feel="$(get_exe plasma-apply-lookandfeel)"; then
       "$look_and_feel" --apply stylix
-    else
-      echo "Skipping plasma-apply-lookandfeel: command not found"
     fi
   '';
-
-  activateDocs = "https://stylix.danth.me/options/hm.html#stylixtargetskdeservice";
+  activator = getExe activator';
 in
 {
-  options.stylix.targets.kde.enable = config.lib.stylix.mkEnableTarget "KDE" true;
+  options.stylix.targets.kde = {
+    enable = mkEnableTarget "KDE" true;
+    decorations = mkOption {
+      type = types.str;
+      default = "org.kde.breeze";
+      description =
+        let
+          url = "https://nix-community.github.io/plasma-manager/options.xhtml#opt-programs.plasma.workspace.windowDecorations.library";
+        in
+        "Custom border decorations. See [plasma-manager docs](${url}).";
+    };
+  };
 
   config =
-    lib.mkIf
-      (config.stylix.enable && cfg.enable && pkgs.stdenv.hostPlatform.isLinux)
+    mkIf (config.stylix.enable && cfg.enable && pkgs.stdenv.hostPlatform.isLinux)
       {
         home = {
           packages = [ themePackage ];
 
           # This activation entry will run the theme activator when the homeConfiguration is activated
-          # Note: This only works in an already running Plasma session.
-          activation.stylixLookAndFeel = lib.hm.dag.entryAfter [ "writeBoundary" ] ''
-            ${lib.getExe activator} || verboseEcho \
-              "KDE theme setting failed. See `${activateDocs}`"
+          activation.stylixLookAndFeel = hm.dag.entryAfter [ "writeBoundary" ] ''
+            ${activator} || verboseEcho \
+              "Stylix KDE theme setting failed. Note that it only works in an already running Plasma session."
```

This might be more concise:

```suggestion
              "Stylix KDE theme setting failed. This only works in a running Plasma session."
```